src/App.tsx

```diff
@@ -8,32 +9,39 @@ function getRandomName(): string {
 }
 
 export const App: React.FC = () => {
-  const today = new Date();
-  let clockName = 'Clock-0';
+  const [showClock, setShowClock] = useState(true);
```

Comment:
it is a boolean value, but it is named like a function
```suggestion
  const [hasClock, setHasClock] = useState(true);
```